test/metabase/test.clj

```diff
@@ -5,27 +5,16 @@
   reduces the cognitive load required to write tests.)"
   (:refer-clojure :exclude [compile])
   (:require
-   [clojure.data]
-   [clojure.test :refer :all]
-   [environ.core :as env]
```

Comment:
I finally moved all the random functions at the bottom of `metabase.test` into their own namespace, something that has been on my TODO list for a long time now.